tools/correction-adopted.py

```
@@ -0,0 +1,242 @@
+        return 2
+
+    if not nums:
+        print("⛔ VOID — zero open issues read. An empty board and a failed query are "
+              "byte-identical here.", file=sys.stderr)
+        return 2
+    if len(nums) < total:
+        print(f"⛔ VOID — read {len(nums)} of {total} stated. A truncated reading cannot "
+              f"support 'no candidates'.", file=sys.stderr)
+        return 2
+
+    print(f"NFORMA-RUN correction-adopted")
```

Ruff (0.16.3)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)